images/capi/ansible/roles/setup/tasks/photon.yml

```
@@ -45,3 +45,16 @@
 - name: install extra RPMs
   command: tdnf install {{ extra_rpms }} -y
   when: extra_rpms != ""
+
+- name: configure auditd rules for containerd
+  template:
+    src: photon-os/etc/audit/rules.d/containerd.rules
+    dest: /etc/audit/rules.d/containerd.rules
+
+# Enable causes auditd to correctly
+# create auto-start symlink.
+# Auditd is diabled by default in Photon OS: https://github.com/vmware/photon/blob/366cf888f3b3c7e54f883c2e3223f5a752fe0e60/SPECS/audit/audit.spec#L104
+- name: configure auditd to start on boot
+  command: "{{ item }}"
+  loop:
+    - systemctl enable auditd
```

I'd recommend using the service module for Ansible, 
```suggestion
- name: Ensure auditd is running and comes on at reboot
  service:
    name: auditd
    state: started
    enabled: yes
```

---

@voor : Thanks for the review. I want to `enable` the service to boot on the start of OS which is different from `starting` the service itself (not required). 
I think I will have to use the `enable` flag and the `state` flag can be anything. I will commit my changes on the next iteration.